shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -726,9 +734,26 @@ \subsection{Protocol Parameter Update Transition}
           \var{oblg_{cur}} & \leteq & \obligation{pp}{stkeys}{stpools}{slot} \\
           \var{oblg_{new}} & \leteq & \obligation{pp_{new}}{stkeys}{stpools}{slot} \\
           \var{diff} & \leteq & \var{oblg_{cur}} - \var{oblg_{new}} \\
-          \\
-          \var{reserves} + \var{diff} & < & 0\\
-      \end{array}}\\
+      \end{array}}
```

Comment:
BTW, shouldn't this section be called "Protocol Parameters Update Transition"?

Reply:
Yes, that 's probably consistent with the rest.

Reply:
oh wait, I misunderstood, you mean "parameters" instead of "parameter", right?

Reply:
Right, aren't we updating a bunch of parameters? But I don't know whether that's the right wording :man_shrugging: :es: